packages/ingest/test/test-HttpsProviderClient.js

```diff
@@ -129,26 +129,26 @@ test('download() downloads a file', async (t) => {
 });
 
 test('sync() downloads remote file to s3 with correct content-type', async (t) => {
-  const bucket = randomString();
-  const key = 'syncedFile.json';
+  const destinationBucket = randomString();
```

Comment:
These changes all just are updating this test to use sync's parameterized call, and updating the variables from generics like `bucket` and `key` to be more consistent.